identity-service/src/notifications/index.js

```diff
@@ -433,9 +433,15 @@ class NotificationProcessor {
       timeout
     )
     const { info: metadata, owners, milestones } = notificationsFromDN
-    const notifications = await filterOutAbusiveUsers(
+    let notifications = await filterOutAbusiveUsers(
       notificationsFromDN.notifications
     )
+
+    // Ensure we don't process any notifs
+    // that are already in the db
+    const latestBlock = await models.Notification.blocknumber
```

Comment:
does this need to be `.max('blocknumber')`?

Reply:
yes. thx braindead today